### PR TITLE
move to go 1.16

### PR DESCRIPTION
**Pull request by giggsoff**
Seems, we need update go version to support `io/fs`.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,8 @@
 # Copyright (c) 2019 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM lfedge/eve-alpine:6.2.0 AS build
-ENV BUILD_PKGS go git
-RUN eve-alpine-deploy.sh
+FROM golang:1.16.3-alpine3.13 AS build
```
Comment (rvs) [inline Dockerfile:4]: Why are you switching to this?

**Review by rvs — CHANGES_REQUESTED** [review]
I don't understand why do we have to switch away from `lfedge/eve-alpine:6.2.0` -- the rest looks fine

**Comment by deitch** [comment]
This pulls it out of using the eve alpine build. I don't mind, but I would hold on this until @rvs can comment. 

I'll look at the rest in the am.

File: Dockerfile (hunk 1 above)
Comment (giggsoff) [reply Dockerfile:4]: With previous notation we have go 1.15.7-r0 (`(13/13) Installing go (1.15.7-r0)`) inside.

**Comment by giggsoff** [comment]
Seems, we have several solutions here:
1. proposed
2. build go from source
3.  change the code to support 1.15

WDYT @deitch @rvs ?

**Comment by rvs** [comment]
I don't understand what is the problem we're trying to solve here @giggsoff

**Comment by giggsoff** [comment]
We cannot build Adam now, you can see it inside GH Action on master: https://github.com/lf-edge/adam/runs/2229612651?check_suite_focus=true#step:6:246

**Comment by deitch** [comment]
I think that the builder is now the official way to go. Is that correct @rvs ? If so, we should update the builder.

**Comment by giggsoff** [comment]
Actually, I can find go 1.16 only in [edge branch](https://pkgs.alpinelinux.org/packages?name=go&branch=edge) of alpine for now.

**Comment by rvs** [comment]
In that case -- I suggest we pull that into the container for now @giggsoff just as an extra RUN step. Meanwhile to @deitch point we can explore bumping the dependency all the way to go 1.16 in all of EVE's project -- can one of you please start that thread on eve-tsc@ ?

**Review by rvs — APPROVED** [review]

**Pull request merged**